spec/models/searchworks_item_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+describe SearchworksItem do
+  let(:item_id) { SearchworksItem.new }
```

Comment:
But if you don't use `subject` you'll still want to send the `has_title?` message to the `@item` variable instead of sending the `has_title?` message to the `@item.success?` boolean.